assem16399/Diva
# feat (signup) ✨ Implement Signup Data Layer.

## Jira Ticket:  [DIVA-59](https://aasemhany.atlassian.net/browse/DIVA-59)


## Description 📖 
Implementing Singup Feature data layer (Models, ApiService, ApiRepo)


## How to Test 📋 
check the Signup Feature Data Layer folder



[DIVA-59]: https://aasemhany.atlassian.net/browse/DIVA-59?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
